### DISCUSSION TITLE
Is it possible to stream both system audio and mic audio without the use of a DAW?

### DISCUSSION THREAD

**Discussion by darrenchang07** [ExistentialAudio/BlackHole#325] [Q&A]
For context, I am trying to fix the issue of sound sharing on Discord. Mac apparently has some restrictions so there is no sound when you screen share with other people. I set up the multi-output device with Garageband, and it works on Discord (my friend can hear both my voice and the youtube video I'm playing on Chrome). However, once I close Garageband my friend can't hear the youtube video anymore. Is it possible for Blackhole to take in both system audio and mic audio without the use of a DAW?

Thanks, and any help is appreciated!

**Comment by devinroth** [comment]
No.

Devin
